query-algebrizer/src/clauses/pattern.rs

```diff
@@ -292,14 +294,18 @@ mod testing {
         SourceAlias,
     };
 
+    fn variable_named(s: &'static str) -> Variable {
```

Comment:
This would definitely be more pleasant if you wrote the transitive closure of `Into`/`From` declarations.  You could end up with `Variable("name".into())` or similar.